test.js

```diff
@@ -139,3 +69,71 @@ test('changes', function (t) {
     t.error(e);
   });
 });
+test('ignore: _attachments', function (t) {
+  t.plan(1);
+  var dbName = 'six';
+  var db = new PouchDB(dbName, {db: memdown});
+  db.crypto('password', {ignore: '_attachments'})
+  db.put({
+    _id: 'id-12345678',
+    _attachments: {
+      'att.txt': {
+        content_type: 'text/plain',
+        data: 'TGVnZW5kYXJ5IGhlYXJ0cywgdGVhciB1cyBhbGwgYXBhcnQKTWFrZS' +
+              'BvdXIgZW1vdGlvbnMgYmxlZWQsIGNyeWluZyBvdXQgaW4gbmVlZA=='
+      }
+    }
+  }).then(function () {
+    return db.get('id-12345678', {
+      attachments: true,
+      binary: true
+    });
+  }).then(function (doc) {
+    t.ok(Buffer.isBuffer(doc._attachments['att.txt'].data), 'returns _attachments as Buffers');
+  })
+  .catch(t.error);
+})
+test('modp: "modp5", ignore: "_attachments"', function (t) {
```

Comment:
I removed the test in https://github.com/calvinmetcalf/crypto-pouch/pull/29/commits/2709adda9cb771ffe2042438791042a99975a9ac